src/components/Info/Info.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+// Take in a phrase and separate the third word in an array
+function createArrayFromPhrase(phrase) {
+  const splitPhrase = phrase.split(' ');
+  const thirdWord = splitPhrase.pop();
+  return [splitPhrase.join(' '), thirdWord];
+}
+
+const InfoSection = props => (
+  <section className={`bx--row ${props.className} info-section`}>
+    <div className="bx--col-md-8 bx--col-lg-4 bx--col-xlg-3">
+      <h3 className="info-section__heading">{props.heading}</h3>
+    </div>
+    {props.children}
+  </section>
+);
+
+const InfoCard = props => {
+  const splitHeading = createArrayFromPhrase(props.heading);
+
+  return (
+    <article className="info-card bx--col-md-4 bx--col-lg-4 bx--col-xlg-3 bx--offset-xlg-1">
```

Comment:
Per the tutorial instructions, this `<article>` tag should have been replaced with a `<div>` tag.

Reply:
Ah you are right, I missed that part.   I added it now.